.nvmrc

```diff
@@ -1 +1 @@
-v12
+16.15.0
```

Comment:
LTS is 16.15.1

Reply:
Whoops, good catch. https://github.com/dr4fters/dr4ft/pull/1622/commits/395d99e6bc1c7dd4d1cc0dd53e4da4165d48319e

Reply:
Does it make sense to not specify the patch version to receive non-breaking fixes?
Like `16.15`. Does that even work?